pkg/version/version_test.go

```diff
@@ -17,6 +18,24 @@ func TestVersionCheck(t *testing.T) {
 	})
 	go http.ListenAndServe("localhost:8080", nil)
```

Comment:
This _definitely_ seems like a problem.  This is an _extremely_ common port.  Would prefer that we at least use a more obscure one (23456, for instance).  Really, however, we shouldn't use ListenAndServe -- we should bind an ephemeral port (i.e. by specifying port 0), take the ephemeral port, and then run the server. Fine to be ticketed for followup